mrrc/config.py

```diff
@@ -37,6 +37,8 @@ class MrrcConfig(object):
     in $HOME/.mrrc/ folder by default.
     """
 
+    __mrrc_configs = dict()
```

Comment:
I don't think this is needed. Instead  we should do checking in _val_or_default. I'll change it later.